ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -78,6 +80,23 @@ def _auth_node(self, node_id, cluster_user, cluster_password):
             Log.error(f"Failed to authenticate node : {node_id} with reason : {e}")
             raise ClusterManagerError(f"Failed to authenticate node : {node_id}, Please check username or password")
 
+    def _get_node_group(self) -> list:
```

Comment:
Could you please modify start code to use this function.

Reply:
Done